lib/naver/searchad/api/core/base_service.rb

```diff
@@ -55,6 +55,13 @@ def apply_command_defaults(command)
             # To be implemented by subclasses
           end
 
+          def validates_presence_of(fields, object)
+            fields.each do |key|
+              raise MissingRequiredAttributeError.new(
```

Comment:
As it's already too long for one line, this one liner doesn't make much sense :D
```
unless object.key?(key)
  raise MissingRequiredAttributeError
    .new("Require #{key} attribute in object")
end
```